example/package.json

```diff
@@ -30,15 +30,16 @@
     "test:prettier": "prettier --check \"./**/*.{js,ts,jsx,tsx,json}\"",
     "prettier:write": "prettier --write \"./**/*.{js,jsx,json,ts,tsx}\"",
     "stylelint:fix": "stylelint \"./**/*.{css,less}\" --formatter verbose --fix",
-    "serve:remote": "serve -s dist/remote -l 4001",
-    "serve:standalone": "serve -s dist/standalone -l 4002",
+    "serve:remote": "serve dist/remote -c serve.json -p 4001",
+    "serve:standalone": "serve dist/standalone -c serve.json -p 4002",
     "eject": "react-scripts eject"
   },
   "dependencies": {
     "ketcher-core": "*",
     "ketcher-macromolecules": "*",
     "ketcher-react": "*",
     "ketcher-standalone": "*",
+    "mui": "^0.0.1",
```

Comment:
This dependency seems irrelevant – it's some old and unsupported package, not related to @mui, I think it can be removed